10a-cw-install.sh

```diff
@@ -5,4 +5,5 @@ set -x
 go get -u github.com/istio-ecosystem/coddiwomple
 cd $GOPATH/src/github.com/istio-ecosystem/coddiwomple
 make deps
-make
+go install ./cmd/cw
```

Comment:
Do you want to make a "make install" makefile target so that we keep the same abstraction everywhere?